lib/mine.js

```diff
@@ -4,26 +4,22 @@
  */
```

Comment:
Agree with the name change but I'd still keep it on the `/` level because the way this module is written it's not really a library imo. Would need to undergo some significant refactoring... 👀